src/main/java/seedu/address/model/booking/Booking.java

```diff
@@ -0,0 +1,62 @@
+package seedu.address.model.booking;
+
+import static seedu.address.commons.util.CollectionUtil.requireAllNonNull;
+
+import java.util.Objects;
+
+import seedu.address.model.person.Person;
+import seedu.address.model.person.Phone;
+
+/**
+ * Represents a Booking in the address book.
+ * Guarantees: details are present and not null, field values are validated, immutable.
+ */
+public class Booking {
+    private final Person person;
+
+    /**
+     * Constructs a booking.
+     *
+     * @param person Person to whom this booking belongs to.
+     */
+    public Booking(Person person) {
+        requireAllNonNull(person);
+        this.person = person;
+    }
+
+    private Person getPerson() {
+        return person;
+    }
+
+    public Phone getPhone() {
+        return getPerson().getPhone();
+    }
+
+    @Override
+    public boolean equals(Object other) {
+        if (other == this) {
+            return true;
+        }
+
+        if (!(other instanceof Booking)) {
+            return false;
+        }
+
+        Booking otherBooking = (Booking) other;
+        return otherBooking.getPerson().equals(getPerson());
```

Comment:
No need. Because for both client and booking, we need to keep modifying the equals (everywhere in fact, even in commands) when we add fields. So we can just make it clear to everyone.

Reply:
@Imranr2 @EuniceSim142 @Manoharan-Ajay-Anand please note that the equals methods of each class need to be updated when you add new stuff!